### ISSUE TITLE
Red color and the hue value=0

### ISSUE THREAD

**Issue by jensr23** [andrei-tatar/node-red-contrib-smartnora#215]
With the spectrumHsv object, the hue value for the color red is 0. Is that correct? Other colors have a value >0.

red = 0 ???
blue = 24
green = 120
yellow = 60
orange = 32.94117736816406
purple = 271.3274230957031
pinc = 349.5238037109375
white = 35.85365676879883
turquoise = 174

![color](https://user-images.githubusercontent.com/116499622/197396521-f82b2d66-1844-4a0c-8532-94f685555bb3.png)

**Comment by andrei-tatar** [comment]
Yes, it's normal. Hue of 0 degrees with full saturation and value, will be red `#FF0000` in hex

![image](https://user-images.githubusercontent.com/6473183/197553062-2c0a494a-4264-458c-bffc-3fa0f9140ffa.png)
